hooks/useMassCalculation.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  calculateTransactionMass,
+  createTransaction,
+  IUtxoEntry,
+} from "@/wasm/core/kaspa";
+import { NetworkType } from "@/contexts/SettingsContext.tsx";
+import { useEffect } from "react";
+
+export default function useMassCalculation(amount: bigint, address?: string) {
```

Comment:
Would it be better to use `IUtxoEntry[]` than amount?

Reply:
To integrate with Broswer API `signAndBroadcastTx`, it would be easier to do it by `IUtxoEntry[]` input.

Reply:
I was not able to use ' IUtxoEntry []` since they are outputs, so I used `IPaymentOutput[].` I am not sure if it fits what you have in mind. Please let me know.